src/components/coin-divider/CoinDivider.js

```diff
@@ -66,6 +70,14 @@ export default function CoinDivider({
     toggleOpenSmallBalances,
   } = useOpenSmallBalances();
 
+  const toggleSmallBalances = useCallback(() => {
+    if (!isSmallBalancesOpen && !fetchedCharts) {
+      dispatch(emitChartsRequest());
+      setFetchedCharts(true);
+    }
+    toggleOpenSmallBalances();
+  }, [dispatch, fetchedCharts, isSmallBalancesOpen, toggleOpenSmallBalances]);
```

Comment:
what happens if you add a new asset e.g. by swapping?

Reply:
worst case scenario it jumps the first time (if network is slow). I'm personally ok with that since it's a very unique case.

Reply:
+1